### PR TITLE
Add select entity to Home Connect

### PR DESCRIPTION
**Pull request by Diegorro98**
## Proposed change

Add select entity information to Home Connect integration documentation

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- This PR is related to the following feature: https://github.com/home-assistant/core/pull/126157

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Enhanced documentation for the Home Connect integration, now detailing support for select entities.
	- Introduced a new "Select" category and platform, allowing users to choose from available programs on devices with programs.

- **Documentation**
	- Clarified functionality regarding program selection in the Home Connect integration documentation.

### CODE REVIEW

File: source/_integrations/home_connect.markdown [hunk 1]
```diff
@@ -32,6 +34,7 @@ The integration will add one Home Assistant device for each connected home appli
 
 - A power switch
 - If the device has programs, switches for each of the individual programs will be added. Note that program options cannot be configured currently.
+- If the device has programs, a select entity that will allow you to select the program between the available ones.
```
Comment (MartinHjelmare) [inline source/_integrations/home_connect.markdown]: We should mention and clarify the two different select entities for each program.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: source/_integrations/home_connect.markdown [hunk 2]
```diff
@@ -31,8 +33,10 @@ The Home Connect integration allows users to integrate their home appliances sup
 The integration will add one Home Assistant device for each connected home appliance which will have the following entities:
 
 - A power switch
-- If the device has programs, switches for each of the individual programs will be added. Note that program options cannot be configured currently.
-- If the device has programs, a timestamp sensor for remaining time and a numeric sensor for the progress percentage.
+- If the device has programs:
+  - Switches for each of the individual programs will be added. Note that program options cannot be configured currently.
```
Comment (Diegorro98) [inline source/_integrations/home_connect.markdown:37]: Should I mark the switches as deprecated once https://github.com/home-assistant/core/pull/131641 gets merged?

Comment (frenck) [reply source/_integrations/home_connect.markdown:37]: Deprecation are generally just removed from the documentation.

**Review by frenck — APPROVED** [review]
Thanks, @Diegorro98 👍 

../Frenck

**Pull request merged**